### DISCUSSION TITLE
Release 3.1.0 shows misleading error message when sending content to Obsidian

### DISCUSSION THREAD

**Discussion by pac-71** [coddingtonbear/obsidian-web#162] [Bug Report]
### Describe the bug

v3.1.0 on edge and brave. 
Error
Could not send content to Obsidian: Error: SyntaxError: Unexpected end of JSON input

### Operating System

Windows 10 22h2 

### Browser

Edge, Brave 

### Browser Version

Microsoft Edge Version 121.0.2277.98 (Official build) (64-bit)
Brave is up to date Version 1.62.156 Chromium: 121.0.6167.139 (Official Build) (64-bit)

**Comment by jack2game** [comment]
I'm getting the same error, `Windows 11` + `Chrome 121.0.6167.140`

**Comment by edwardtoday** [comment]
I start to see the same error message today.

extension ver: 3.1.0
browser ver: Chrome 121.0.6167.139 / Arc 1.28.0 (45831)

What should be noted is that the markdown clippings has been added successfully to my daily note every time I use this extension, regardless of the error messages.

**Reply by pac-71** [reply]
You are right. I had a bunch of repeated notes that I just assumed were not being put given the error message.

**Comment by stoweboyd** [comment]
Me too

**Comment by coddingtonbear** [comment]
I've just pushed up a new build to the Chrome App store (v3.1.1) that should fix this up for you all.  I can't say how long the review process will take, but the most recent update made it through the Chrome app store review process in just a few hours.

**Comment by LuchianC** [comment]
Version - 3.1.1 it's up, everything it's fine now, Thanks
